docs-ref-conceptual/azure-cli-sp-tutorial-1.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Work with Azure service principals – Azure CLI | Microsoft Docs
+description: Learn how to create and use service principals with the Azure CLI. Use service principals to gain control over which Azure resources can be accessed.
+manager: jasongroce
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 08/2/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli, seo-azure-cli
+keywords: azure service principal, create service principal azure, create service principal azure cli
+---
+# 1 - Work with Azure service principal using the Azure CLI
+
+Automated tools that use Azure services should always have restricted permissions, to ensure that your resources will be secure. Therefore, instead of having applications sign in as a fully privileged user, Azure offers service principals.
+
+## What is an Azure service principal?
+
+An Azure service principal is an identity created for use with applications, hosted services, and automated tools, to access resources. For security reasons, it's always recommended to use service principals with automated tools rather than allowing them to sign in with a user identity.
+
+In this tutorial, you'll learn how to use service principals with either password-based or certificate-based authentication in order to create a resource. The roles assigned to the service principal restrict access, which gives you control over which resources can be accessed and at what level.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+> * Use service principals with a password.
+> * Use service principals with a certificate.
+> * Get an existing service principal.
+> * Manage service principal roles
+> * Create a resource using service principal
+
+## Prerequisites 
```

Comment:
Prerequisites need to follow the azCLI [guidelines](https://review.learn.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=main#prerequisites).  Here is another example from inside this same repo: https://review.learn.microsoft.com/en-us/cli/azure/azure-cli-vm-tutorial-1?branch=pr-en-us-3798#prerequisites. It is also important to talk about the permissions needed in this prerequisite section.

Your markdown would look like this:

## Prerequisites
* To create a service principal, you must have ... <what permissions?>
[!INCLUDE [include](~/articles/reusable-content/azure-cli/azure-cli-prepare-your-environment-no-header.md)]